x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -215,6 +224,19 @@ export const links: LinkItem = {
       hideTimeline: true,
     },
     cloudDefendLink,
+    {
+      id: SecurityPageName.notes,
+      title: NOTES,
+      description: i18n.translate('xpack.securitySolution.appLinks.notesDescription', {
+        defaultMessage:
+          'Oversee, revise and revisit the annotations within each document and timeline.',
```

Comment:
This is the same text as for `xpack.securitySolution.navLinks.investigations.note.title`. Do you think it makes sense to use just one of them? Or do you see these two diverge in the future?

Reply:
it is indeed the same but I decided to keep them separate as the old navigation will eventually (hopefully soon) be removed and I wanted to make that cleanup as easy as possible. We won't have to remember to go remove some translation somewhere... everything is collocated...
If you have a strong opinion I'm not against using a single one, I just feel like here it's unnecessary